help.go

```diff
@@ -26,6 +35,15 @@ func (e *Executor) PrintTasksHelp() {
 	w.Flush()
 }
 
+func (e *Executor) taskNames() (tasks []*taskfile.Task) {
```

Comment:
`taskNames` -> `allTaskNames`

Reply:
resolved in https://github.com/therealkevinard/task/commit/42702e81b35c5b4960c94ff65577f6c4314b0ccd